File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
    int n = nums.length;
    ans = new int[2*n];
    ans = nums;
    for(int i=0;i<n;i++)
    {
    if(ans[i]==nums[i])
    {
     ans[i+n]=num[i];
    }
    }
    return ans;
    

        
    }
}

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:

        #bubble sort 
        for i in range(len(nums)-1):
            for j in range(len(nums)-i-1):
                if nums[j]>nums[j+1]:
                    temp = nums[j]
                    nums[j]=nums[j+1]
                    nums[j+1]=temp
        return nums


        